src/eviction.cpp

```diff
@@ -302,6 +302,22 @@ void EvictionManagerImpl::UpdateRelayTxs(NodeId id, bool relay_txs)
     }
 }
 
+void EvictionManagerImpl::AddBlockInFlight(NodeId id)
+{
+    LOCK(m_candidates_mutex);
+    if (const auto& it = m_candidates.find(id); it != m_candidates.end()) {
+        ++it->second.m_blocks_in_flight;
```

Comment:
This is really confusing to read.

Instead of trying to convince myself that it's correct, I'll ask that you make it more obviously correct:`it->second.m_blocks_in_flight++`

Reply:
Good point. The dev docs recommend this way but in this case its obviously better to do `foo++`.